[2024-07-24T12:28:19.096+0900] {processor.py:161} INFO - Started process (PID=86499) to work on /Users/DE32/.pyenv/versions/3.11.9/envs/note/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-24T12:28:19.096+0900] {processor.py:830} INFO - Processing file /Users/DE32/.pyenv/versions/3.11.9/envs/note/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-24T12:28:19.096+0900] {logging_mixin.py:188} INFO - [2024-07-24T12:28:19.096+0900] {dagbag.py:545} INFO - Filling up the DagBag from /Users/DE32/.pyenv/versions/3.11.9/envs/note/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-24T12:28:19.109+0900] {logging_mixin.py:188} INFO - [2024-07-24T12:28:19.109+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/DE32/.pyenv/versions/3.11.9/envs/note/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-24T12:28:19.109+0900] {logging_mixin.py:188} INFO - [2024-07-24T12:28:19.109+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-24T12:28:19.109+0900] {processor.py:842} WARNING - No viable dags retrieved from /Users/DE32/.pyenv/versions/3.11.9/envs/note/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-24T12:28:19.136+0900] {processor.py:183} INFO - Processing /Users/DE32/.pyenv/versions/3.11.9/envs/note/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2024-07-24T12:29:43.850+0900] {processor.py:161} INFO - Started process (PID=86707) to work on /Users/DE32/.pyenv/versions/3.11.9/envs/note/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-24T12:29:43.850+0900] {processor.py:830} INFO - Processing file /Users/DE32/.pyenv/versions/3.11.9/envs/note/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-24T12:29:43.850+0900] {logging_mixin.py:188} INFO - [2024-07-24T12:29:43.850+0900] {dagbag.py:545} INFO - Filling up the DagBag from /Users/DE32/.pyenv/versions/3.11.9/envs/note/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-24T12:29:43.863+0900] {logging_mixin.py:188} INFO - [2024-07-24T12:29:43.863+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/DE32/.pyenv/versions/3.11.9/envs/note/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-24T12:29:43.863+0900] {logging_mixin.py:188} INFO - [2024-07-24T12:29:43.863+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-24T12:29:43.863+0900] {processor.py:842} WARNING - No viable dags retrieved from /Users/DE32/.pyenv/versions/3.11.9/envs/note/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-24T12:29:43.890+0900] {processor.py:183} INFO - Processing /Users/DE32/.pyenv/versions/3.11.9/envs/note/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
